File: Matrix/codeforces_263A.py

# Problem link ===>> https://codeforces.com/problemset/problem/263/A

matrix = []
row, col = 0, 0

# Read the matrix
for i in range(5):
    row_values = list(map(int, input().split()))
    matrix.append(row_values)

    # Check if the current row contains 1
    if 1 in row_values:
        row = i
        col = row_values.index(1)

# Calculate the minimum number of moves to center
moves = abs(row - 2) + abs(col - 2)

# Output the result
print(moves)

File: Matrix/codeforces_263A.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/263/A
// submission link ===>> https://codeforces.com/contest/263/submission/209298457

#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    int matrix[5][5];
    int row, col;

    // Read the matrix
    for (int i = 0; i < 5; i++)
    {
        for (int j = 0; j < 5; j++)
        {
            cin >> matrix[i][j];

            // Check if the current element is 1
            if (matrix[i][j] == 1)
            {
                // Calculate the minimum number of moves to center
                row = abs(i - 2);
                col = abs(j - 2);
            }
        }
    }

    // Calculate the total number of moves to center
    int moves = row + col;

    // Output the result
    cout << moves << endl;

    return 0;
}
